### PR TITLE
Nitrpg work

### PR DESCRIPTION
**Pull request by privat**
Improve the dashboard on the player page with `request_for_comment` things, and assigned work

### CODE REVIEW

**Comment by Morriar** [comment]
Where is the demo?

File: contrib/nitrpg/src/templates/panels.nit [hunk 1]
```diff
@@ -360,16 +360,67 @@ class PlayerReviewsPanel
...
 
-		var issues = game.repo.search_issues(q)
+		var q2 = "is:open label:request_for_comments sort:updated-asc " +
+			"-involves:{player.name}"
+
+		var issues = new ArraySet[Issue]
+		issues.add_all game.repo.search_issues(q).as(not null)
+		issues.add_all game.repo.search_issues(q2).as(not null)
+		if issues.is_empty then
+			add "<em>No pull request or issues to review yet...</em>"
```
Comment (Morriar) [inline contrib/nitrpg/src/templates/panels.nit]: or issue*

File: contrib/nitrpg/src/templates/panels.nit [hunk 2]
```diff
@@ -360,16 +360,67 @@ class PlayerReviewsPanel
...
-		add "Review pull requests to gain nitcoins!"
+		add "Review pull requests and comment issues to gain nitcoins!"
 	end
 
 	redef fun render_body do
 		var q = "is:open label:need_review sort:updated-asc " +
 			"-involves:{player.name}"
 
-		var issues = game.repo.search_issues(q)
+		var q2 = "is:open label:request_for_comments sort:updated-asc " +
```
Comment (Morriar) [inline contrib/nitrpg/src/templates/panels.nit]: I really need to add a config file to set all this Nit related labels.

**Comment by Morriar** [comment]
This seems cool but I want to see a demo!

**Comment by privat** [comment]
@Morriar could you merge this PR when you will setup the next demo for your PR on nitrpg?

**Comment by privat** [comment]
reroll: spelling

**Comment by Morriar** [comment]
Demo is up: http://nitlanguage.org/rpg/games/privat/nit/players/privat

**Comment by Morriar** [comment]
Hum... maybe you should filter your own issues from the list?

**Comment by Morriar** [comment]
I'm really not sure of your panel. It adds a lot of noise in the page. Also you should split pr reviews and issue comment

Plus, because nitrpg actually gives nothing (for now) when you fixe an issue, it's not that true that you can "gain nitcoins by doing your tasks".

But +1 anyway

**Comment by privat** [comment]
in fact, nitrpg may reward you for random comments: with the achievements `from lurker to member`, or `ping god`.

**Comment by Morriar** [comment]
Yes plus the fact you gain a certain amount of nitcoin for each comment, I forgot that. So +1

**Pull request merged**